import-maps/static-import.py

```diff
@@ -0,0 +1,2 @@
+def main(request, response):
+    return 'import "{}";\n'.format(request.GET.first('url'))
```

Comment:
HTTP response header `Content-Type: application/javascript` or so should be added. (Probably this is why bots are failing)